src/AppBundle/Entity/Relay.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace AppBundle\Entity;
+
+use DateTime;
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
+
+/**
+ * A contributor can relay a notice from another contributor. This entity represents this relation.
+ *
+ * @ORM\Entity
+ * @ORM\Table(name="relay")
+ * @UniqueEntity(
+ *     fields={"contributor", "notice"},
+ *     errorPath="contributor",
+ *     message="This contributor already relayed that notice."
+ * )
+ */
+class Relay
+{
+    /**
+     * @var Contributor
+     *
+     * @ORM\ManyToOne(targetEntity=Contributor::class, inversedBy="relayedNotices", fetch="EAGER")
+     * @ORM\JoinColumn(nullable=false)
+     * @ORM\Id
+     */
+    private $contributor;
```

Comment:
I'm tempted to rename this property `$relayedBy` to rule out any possible ambiguity.

Reply:
Done.